rstudio/packrat
# Update vendored `renv` with `renv:::vendor()`

Update vendored `renv` to the latest version — the latest version on GitHub is `0.17.3-62`.

Since the last time we updated it, `renv` now includes an `renv:::vendor()` method, so I used this. I removed the old script we used to use.

This update is to make available the PPM repo normalization methods available in Packrat.

### Manual Testing Notes

To confirm that bundled `renv` functions still work as expected, deploy this shiny app to a Connect instance:

```r
library(shiny)

ui <- fluidPage(
    titlePanel("Old Faithful Geyser Data"),
    sidebarLayout(
        sidebarPanel(
            sliderInput("bins",
                        "Number of bins:",
                        min = 1,
                        max = 50,
                        value = 30),
            div("username", verbatimTextOutput("user")),
            div("groups", verbatimTextOutput("groups"))
        ),

        mainPanel(
            plotOutput("plot")
        )
    )
)

server <- function(input, output, session) {
  output$plot <- renderPlot({
    x    <- faithful[, 2]
    bins <- seq(min(x), max(x), length.out = input$bins + 1)

    hist(x, breaks = bins, col = 'darkgray', border = 'white')
  })

  output$user <- renderText(session$user)
  output$groups <- renderText(session$groups)
}

shinyApp(ui = ui, server = server)
```
